packages/sdk/src/util.test.ts

```diff
@@ -206,14 +206,44 @@ export async function setupWalletsAndContexts() {
     const aliceSpaceDapp = createSpaceDapp(aliceProvider, baseConfig.chainConfig)
     const carolSpaceDapp = createSpaceDapp(carolProvider, baseConfig.chainConfig)
 
+    // Use this wrapper around the space dapp to help the client correctly evaluate
+    // on-chain entitlements for channel operations.
+    class SpaceDappChannelEntitlementsDelegate {
```

Comment:
I added an entitlements class here https://github.com/river-build/river/blob/d731364051a4600c1bc4157d6c0cc25a6365d203/packages/sdk/src/sync-agent/entitlements/entitlements.ts#L5

can we use that instead? I want us to flush this out with the wallet linking code that needs to get ported from harmony

Reply:
No problem, TY!